modules/ovn-kubernetes-logical-architecture-con.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: CONCEPT
+[id="ovn-kubernetes-logical-architecture-con_{context}"]
+= OVN-Kubernetes logical architecture
+
+The following diagram shows the OVN-Kubernetes logical architecture.
+
+.OVN-Kubernetes architecture
+image::299_OpenShift_OVN-Kubernetes_arch_0123_2.png[OVN-Kubernetes logical architecture]
```

Comment:
This alt text doesn't really describe the image.

Here are some outside examples that might be helpful:

- https://www.w3.org/WAI/tutorials/images/complex/
- https://sc.edu/about/offices_and_divisions/digital-accessibility/guides_tutorials/alternative_text/charts-diagrams/index.php ("Example of Describing a Diagram")

Ignore the longdesc parts in the w3 site--there's no great way to do it with our toolchain, anyway, but "Bar chart showing blah blah blah" is fair.